includes/Stories_Renderer/Carousel_Renderer.php

```diff
@@ -80,33 +81,54 @@ public function render( array $args = [] ) {
 		}
 
 		parent::render( $args );
+		$block_classes     = $this->get_block_classes();
 		$container_classes = $this->get_container_classes();
-		$container_style   = $this->get_container_styles();
 
 		ob_start();
 		?>
-		<div>
-			<div
-				class="<?php echo esc_attr( $container_classes ); ?>"
-				style="<?php echo esc_attr( $container_style ); ?>"
-			>
-				<amp-carousel
-					width="1"
-					height="1"
-					layout="intrinsic"
-					type="carousel"
-					role="region"
-					aria-label="<?php esc_attr_e( 'Web Stories', 'web-stories' ); ?>"
-				>
+		<div class="<?php echo esc_attr( $container_classes ); ?>">
+			<div class="<?php echo esc_attr( $block_classes ); ?>">
+				<?php
+				if ( ! $this->is_amp_request() ) {
+					?>
+					<div class="web-stories-list__carousel">
+						<?php
+						foreach ( $this->story_posts as $story ) {
+							$this->render_single_story_content();
+							$this->next();
+						}
+						$this->maybe_render_archive_link_card();
+						?>
+					</div>
+					<div tabindex="-1" aria-label="Previous" class="glider-prev">←</div>
+					<div tabindex="0" aria-label="Next" class="glider-next">→</div>
+					<div role="tablist" class="dots"></div>
 					<?php
-					foreach ( $this->story_posts as $story ) {
-						$this->render_single_story_content();
-						$this->next();
-					}
+					$this->render_stories_with_lightbox_noamp();
+				}
+				if ( $this->is_amp_request() ) {
 					?>
-				</amp-carousel>
+					<amp-carousel
+						width="1"
+						height="1"
+						layout="intrinsic"
+						type="carousel"
+						role="region"
+						aria-label="<?php esc_attr_e( 'Basic carousel', 'web-stories' ); ?>"
```

Comment:
```suggestion
						aria-label="<?php esc_attr_e( 'Web Stories', 'web-stories' ); ?>"
```

Reply:
@divyarajmasani Please do not commit these changes as those are already performed in [other PR](https://github.com/rtCamp/web-stories-wp/pull/152/commits/ba9be31508ce35b548fecbdab9170364b570f97a)

Reply:
noted,